src/couch/src/couch_db_int.hrl

```diff
@@ -32,6 +32,8 @@
     before_doc_update = nil, % nil | fun(Doc, Db) -> NewDoc
     after_doc_read = nil,    % nil | fun(Doc, Db) -> NewDoc
 
+    % feature removed in 3.x, but field kept to avoid changing db record size
+    % and breaking rolling cluster upgrade
     waiting_delayed_commit = nil,
```

Comment:
You might rename this field which will cause the compiler to complain if its referneced anywhere that was missed. Something simple like `waiting_delayed_commit_deprecated` or similar.

Reply:
Good call